src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -410,6 +411,7 @@ def _get_version_string(name, version_dict):
     _print()
 
     _print("Python location '{}'".format(os.path.abspath(sys.executable)))
+    _print("Config directory '{}'".format(get_config_dir()))
```

Comment:
Calling `get_config_dir()` is the official way of retrieving Azure CLI's config dir. `azure.cli.core.auth.identity.Identity` also uses it:

https://github.com/Azure/azure-cli/blob/6a1c3ca773310263a8a1bf48a370e84e6127b471/src/azure-cli-core/azure/cli/core/auth/identity.py#L85